### DISCUSSION TITLE
ランキングメールのURLがそのままだとシェアしづらい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#410] [要望・アイデア・不具合]
毎週水曜に届くランキングメールのリンクから記事に飛ぶと、URLにいろいろとパラメータが付きます。

https://qiita.com/yamaguchi2000/items/76060c08764ce4c704f9?utm_source=Qiita%E3%83%8B%E3%83%A5%E3%83%BC%E3%82%B9&utm_campaign=9881528a8b-Qiita_newsletter_546_12_21_2022&utm_medium=email&utm_term=0_e44feaa081-9881528a8b-22103397

「この記事、みんな見てみてー」とシェアしたいと思っても、無駄にURLが長すぎるので、つい手でパラメータを消してしまいます。

https://qiita.com/yamaguchi2000/items/76060c08764ce4c704f9

Twitterでシェアするときも同様にパラメータが付きます。

<img width="662" alt="Screenshot 2023-01-04 at 7 05 02" src="https://user-images.githubusercontent.com/1148320/210451747-73f75da1-0c08-4e1d-833f-1ec8f5b3b5f5.png">

あと、僕は「はてなブックマーク」のChrome拡張を入れているのですが、パラメータが含まれているとブックマーク数を確認できません。

<img width="511" alt="Screenshot 2023-01-04 at 7 05 28" src="https://user-images.githubusercontent.com/1148320/210451933-275a65ba-8b02-4270-affb-e920ede0dc7e.png">

ブックマーク数を見るためにはパラメータなしのURLに切り替える必要があります。
<img width="515" alt="skitch" src="https://user-images.githubusercontent.com/1148320/210452023-b20e1051-e08d-43a5-8731-04dc4a526042.png">

この問題は以前から地味に気になっていました。
ランキングメールからアクセスしたときはパラメータなしのURLにリダイレクトするなど、パラメータの影響を受けない形に変更してもらえるとありがたいです。

### 参考情報
たとえばRubyWeeklyというメルマガだと、メルマガ上のURLは https://rubyweekly.com/link/133475/web ですが、クリックすると  https://github.com/rails/docked にリダイレクトするような仕組みになっています。

参考 https://rubyweekly.com/issues/634

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ありがとうございます！🙏
他にも同様のご意見はいただいているので、こちらも合わせて検討させていただきます！
